<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>551</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Image Result</string>
       </property>
       <widget class="QLabel" name="imageshow_result">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>251</width>
          <height>341</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>Result Image</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Image Raw</string>
       </property>
       <widget class="QLabel" name="imageshow_raw">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>251</width>
          <height>341</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>Raw Image</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>301</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Capture control</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_start">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_stop">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Start/Stop</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_blink">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable Blink Detect</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>150</y>
      <width>301</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Eye tracking Result</string>
    </property>
    <widget class="QLabel" name="imageshow_eye">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>251</width>
       <height>161</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Eye Image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>CaptureEye</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>880</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionLoad_Video"/>
    <addaction name="actionSave_Data"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Mode">
    <property name="title">
     <string>&amp;Mode</string>
    </property>
    <addaction name="action_Online_Mode"/>
    <addaction name="action_Video_Mode"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Mode"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad_Video"/>
   <addaction name="actionSave_Data"/>
   <addaction name="separator"/>
   <addaction name="action_Online_Mode"/>
   <addaction name="action_Video_Mode"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_Video">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/fileopen.png</normaloff>:/icons/icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Load Video</string>
   </property>
  </action>
  <action name="actionSave_Data">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/filesave.png</normaloff>:/icons/icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save Data</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Online_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/gnome-lockscreen.png</normaloff>:/icons/icons/gnome-lockscreen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Online Mode</string>
   </property>
  </action>
  <action name="action_Video_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/document-open-recent.png</normaloff>:/icons/icons/document-open-recent.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Video Mode</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
